cola/widgets/main.py

```diff
@@ -76,6 +76,10 @@ def __init__(self, model, parent=None, settings=None):
         # Runs asynchronous tasks
         self.runtask = qtutils.RunTask()
 
+        from cola.widgets.toolbar import create_toolbar
```

Comment:
If I put with the others it gives me the following error with all the icons.

`Cannot open file '/home/ubuntu/python/git-cola/bin/icons:folder-new.svg', because: No such file or directory`